tests/integration/test_active_response/test_analysisd/test_os_exec.py

```diff
@@ -232,10 +232,16 @@ def get_configuration(request):
 @pytest.fixture(scope="function")
 def restart_service():
     """Restart the Wazuh manager and clean the ossec.log file."""
-    control_service('stop')
-    clean_logs()
-    control_service('start')
-    yield
+    manager_restart_failure = False
+    try:
+        control_service('stop')
+        clean_logs()
+        control_service('start')
+    except Exception:
+        print("Expected exception occurred")
+        manager_restart_failure = True
+
+    yield manager_restart_failure
```

Comment:
This change is not required, due to this fixture do not launch simulator threads.

Reply:
Change done.